src/app/common/ng-fullcalendar/ng-full-calendar.component.ts

```diff
@@ -129,13 +129,14 @@ export class NgFullCalendarComponent implements AfterViewInit {
         };
         this.options.eventRender = function (event, element, view) {
             let detail: RenderEventModel = { event: event, element: element, view: view };
-          let widgetEvent = new CustomEvent('eventRender', {
+            let widgetEvent = new CustomEvent('eventRender', {
                 bubbles: true,
                 detail: detail
             });
             for (let i = 0; i < elem.length; i++) {
                 elem[i].dispatchEvent(widgetEvent);
             }
+            element.css('overflow-y', 'visible');
```

Comment:
why not in css file but inline in code? if there is no class in css already there then why not to just create a css class and assign the class here to the element, where the specifics what this class is to be in css file rather than here inline in js.

Reply:
@andrewtaylorkainos please take a look at calendar.component.ts::114 - this might be an issue.
With css class, you can try put it in calendar styles. This might help https://stackoverflow.com/a/36265072
If this doesn't work you can set global css in style.css

Reply:
So, the problem is with calendar.component.ts::114.

In ie11, this is what causes the element to expand.
I can restrict this by setting a height 

on the element, although we would still be left with the scrollbar (see attached image).

![virtualbox_msedge - win10_26_10_2018_15_19_23](https://user-images.githubusercontent.com/20964997/47568932-a6831200-d932-11e8-82b8-f6113d3ec51f.png)

As far as I'm aware, @radoslawlandowskikainos had 'overflow-y: auto' so that users can scroll to see other hearings within the session.

Should I leave the scrollbar as it is, or include styling to improve it?

Reply:
the scroll: visible makes it impossible to read. The option without overflow y seems to the best option

Reply:
I have removed the change entirely and the CSS changes on line calendar.component.ts::114